content/client-lib-development-guide/features.textile

```diff
@@ -479,7 +480,7 @@ h3(#realtime-channel). Channel
 * @(RTL11)@ If a channel enters the @DETACHED@, @SUSPENDED@ or @FAILED@ state, then all presence actions that are still queued for send on that channel per "RTP16b":#RTP16b should be deleted from the queue, and any callback passed to the corresponding presence method invocation should be called with an @ErrorInfo@ indicating the failure
 ** @(RTL11a)@ For clarity, any messages awaiting an @ACK@ or @NACK@ are unaffected by channel state changes i.e. a channel that becomes detached following an explicit request to detach may still receive an @ACK@ or @NACK@ for messages published on that channel later
 * @(RTL4)@ @Channel#attach@ function:
-** @(RTL4a)@ If already @ATTACHED@ nothing is done
+** @(RTL4a)@ If already @ATTACHED@ @ATTACH@ ProtocolMessage is generated and sent using the most recent @ChannelOptions@. Ably (the server) should send the first message after the atach point in full and not as delta message if delta subscription is active.
```

Comment:
Typo, `atach` is `attach`

Reply:
@mattheworiordan you might be looking at older commits. This text has been gone for a while.